### DISCUSSION TITLE
Why Neovim is v0.10+ when latest release is v0.9.4?

### DISCUSSION THREAD

**Discussion by Andrew15-5** [mikesmithgh/kitty-scrollback.nvim#53] [Q&A] [answered]
Version 0.10.0 is still in development. Why it is required? Does v0.9.4 won't work?

**Comment by Andrew15-5** [comment]
```
Error detected while processing /tmp/kitty-scrollback.nvim.tmp/nvim/s
ite/pack/vendor/start/kitty-scrollback.nvim/scripts/init.lua:
E5113: Error while calling lua chunk: ...-scrollback.nvim/lua/kitty-s
crollback/kitty_commands.lua:343: attempt to call field 'system' (a n
il value)
stack traceback:
        ...-scrollback.nvim/lua/kitty-scrollback/kitty_commands.lua:3
43: in function 'try_detect_nerd_font'
        ...pack/vendor/start/kitty-scrollback.nvim/scripts/init.lua:9
: in main chunk
kitty-scrollback.nvim: Fatal error, on version NVIM 0.9.4. Neovim ver
sion 0.10 or greater is required to work with kitty-scrollback.nvim
[Q]uit, (C)ontinue:
```

`:h news`:
```
==============================================================================
NEW FEATURES                                                    *news-features*

• |vim.system()| for running system commands.
```

`:h vim.system()`:

```
vim.system({cmd}, {opts}, {on_exit})                            *vim.system()*
    Runs a system command or throws an error if {cmd} cannot be run.

    Parameters:  
      • {cmd}      (string[]) Command to execute
```

Wow, this is indeed the feature that was missing in Neovim (that I wanted to have), and it is very useful, because it's easier to pass unquoted arguments with special symbols etc.

**Reply by mikesmithgh** [reply]
Hey @Andrew15-5, yep v0.10 is required for kitty-scrollback.nvim. As you already found 🙂, I am using `vim.system` which is easier to work with than `vim.fn.jobstart` or `vim.fn.system`. Since this plugin never released on v0.9, I just made it require v0.10 to avoid worrying about backward compatibility.

**Reply by Andrew15-5** [reply]
@mikesmithgh, I just found out that there is a `vim.uv.spawn()` (or `vim.loop.spawn`) in Neovim (separate string for command and each arguments), and it looks like it was available a long time ago. See `:h vim.uv` and `:h uv.spawn()`. If it is an old API, then maybe you should consider at least going down to v0.9.0 or v0.9.4 (latest non-dev release). Because I feel not everyone wants to use a dev build as it can potentially cause many problems and not everyone would want such side effects only to use this plugin.

**Reply by mikesmithgh** [reply]
Thanks @Andrew15-5 !

I created the issue https://github.com/mikesmithgh/kitty-scrollback.nvim/issues/58 to track this. Please upvote or comment on that issue so I can gauge interest. 👍

**Comment by mikesmithgh** [comment, marked as answer]
@Andrew15-5 kitty-scrollback.nvim now supports neovim `v0.9` 😸
